flink-yarn/src/main/java/org/apache/flink/yarn/AbstractYarnClusterDescriptor.java

```diff
@@ -745,6 +745,17 @@ public ApplicationReport startAppMaster(
 		// ship list that enables reuse of resources for task manager containers
 		StringBuilder envShipFileList = new StringBuilder();
 
+		int fileReplication = yarnConfiguration.getInt(DFSConfigKeys.DFS_REPLICATION_KEY, DFSConfigKeys.DFS_REPLICATION_DEFAULT);
```

Comment:
following @tillrohrmann 's comment, should we use `yarn.sharedcache.nm.uploader.replication.factor ` instead of the `dfs.replication` here? see: https://hadoop.apache.org/docs/r2.8.5/hadoop-yarn/hadoop-yarn-common/yarn-default.xml